src/main/java/uk/gov/hmcts/juror/api/moj/controller/JurorRecordController.java

```diff
@@ -434,6 +436,39 @@ public ResponseEntity<JurorAttendanceDetailsResponseDto> getJurorAttendanceDetai
         return ResponseEntity.ok().body(details);
     }
 
+
+    /**
+     * Get the attendance and payments log details for a given Juror.
+     *
+     * @param jurorNumber Unique Juror number of the juror
+     * @return Fully populated DTO of the juror's attendance and payments log
+     */
+    @GetMapping(path = "/{jurorNumber}/payments")
+    @Operation(summary = "Get attendance and payments log for a given juror")
+    public ResponseEntity<JurorPaymentsResponseDto> getJurorPaymentsHistory(
+        @PathVariable("jurorNumber") @Valid @JurorNumber
+        @Parameter(description = "Valid juror number", required = true) String jurorNumber) {
+
+        return ResponseEntity.ok().body(jurorRecordService.getJurorPayments(jurorNumber));
+    }
+
+    /**
+     * Get the history log for a given Juror.
+     *
+     * @param jurorNumber Unique Juror number of the juror
+     * @return Fully populated DTO of the juror's history entries
+     */
+    @GetMapping(path = "/{jurorNumber}/history")
+    @Operation(summary = "Get history log for a given juror")
+    public ResponseEntity<JurorHistoryResponseDto> getJurorHistory(
```

Comment:
Do we need to restrict to court/bureau user types? use the annotation like IsCourt

Reply:
Bureau needs this while they own the juror so I've not added this.